docker/postgis/Dockerfile

```diff
@@ -95,4 +106,4 @@ RUN set -eux  ;\
 
 # The script should run after the parent's 10_postgis.sh runs
 # so it must have the name that's listed after that.
-COPY ./initdb-postgis.sh /docker-entrypoint-initdb.d/20_omt_postgis.sh
+COPY ./initdb-postgis.sh /docker-entrypoint-initdb.d/20_omt_postgis.sh
```

Comment:
any reason why you are deleting trailing newline?  I think your editor needs to be configured to always add it -- makes future diffing much easier.

Reply:
Thanks, I add it.